onboard-server/src/main/java/com/objectcomputing/checkins/services/onboard/onboardeeprofile/OnboardingProfileRepository.java

```diff
@@ -9,50 +10,73 @@
 
 import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
-import java.util.Optional;
 import java.util.UUID;
 
-@JdbcRepository (dialect = Dialect.POSTGRES)
+@JdbcRepository(dialect = Dialect.POSTGRES)
 public interface OnboardingProfileRepository extends ReactorCrudRepository<OnboardingProfile, UUID> {
 
-    Optional<OnboardingProfile> findBySocialSecurityNumber(@NotNull String socialSecurityNumber);
+        @Override
+        @Query(value = "SELECT onboard_profile_id, " +
+                        "PGP_SYM_DECRYPT(cast(mp.firstName as bytea),'${aes.key}') as firstName, " +
+                        "PGP_SYM_DECRYPT(cast(mp.middleName as bytea),'${aes.key}') as middleName," +
+                        "PGP_SYM_DECRYPT(cast(mp.lastName as bytea),'${aes.key}') as lastName," +
+                        "PGP_SYM_DECRYPT(cast(mp.socialSecurityNumber as bytea),'${aes.key}') as socialSecurityNumber,"
+                        +
+                        "PGP_SYM_DECRYPT(cast(mp.birthDate as bytea),'${aes.key}') as birthDate," +
+                        "PGP_SYM_DECRYPT(cast(mp.currentAddress as bytea),'${aes.key}') as currentAddress," +
+                        "PGP_SYM_DECRYPT(cast(mp.previousAddress as bytea),'${aes.key}') as previousAddress," +
+                        "PGP_SYM_DECRYPT(cast(mp.phoneNumber as bytea),'${aes.key}') as phoneNumber," +
+                        "PGP_SYM_DECRYPT(cast(mp.secondPhoneNumber as bytea),'${aes.key}') as secondPhoneNumber," +
+                        "PGP_SYM_DECRYPT(cast(mp.personalEmail as bytea),'${aes.key}') as personalEmail " +
+                        "FROM \"onboard_profile\" mp " +
+                        "WHERE (onboard_profile_id = :onboardProfileId) ", nativeQuery = true)
```

Comment:
is this supposed to be (and I don't know) "WHERE (onboard_profile_id = :id) "?

Reply:
findById(@NotNull UUID onboardProfileId) defines value name as onboardProfileId so "WHERE (onboard_profile_id = :onboardProfileId) " is checking for where the id in the table is equal the value being passed through

Reply:
ah, makes sense. thanks for clarifying :)